OpenXmlPowerTools/DocumentBuilder.cs

```diff
@@ -1959,6 +2728,10 @@ private class FromPreviousSourceSemaphore { };
             var ipp2 = oldContentPart.Parts.FirstOrDefault(ipp => ipp.RelationshipId == relId);
             if (ipp2 != null)
             {
+                var oldPart2 = ipp2.OpenXmlPart;
+                if (!(oldPart2 is ImagePart))
```

Comment:
all these casts here can also use pattern matching to decrease the multiple casts currently in use

Reply:
I agree with your comment, but I'd like to delay making this change. My reasoning is that if I make such changes, I probably would want to redo the entire bulk test, which takes more than 24 hours running time, and right now I don't have the time for that.  This code predates pattern matching.